packages/testing/src/temp.ts

```diff
@@ -0,0 +1,16 @@
+import { mkdirSync, realpathSync } from 'fs'
+import { tmpdir } from 'os'
+import { normalize } from 'path'
+
+/** Create and retrieve a new temporary sub-directory */
+export const getTempDir = (): string => {
+  const tempDir = getTempName()
```

Comment:
Consider renaming this variable as it is easy to confuse with the imported `tmpdir`. Confusing the two can be hair pulling to debug.